assembl/static2/js/app/utils/context.js

```diff
@@ -0,0 +1,11 @@
+import LangString from './langString';
+
+class Context {
```

Comment:
Hmmm. Good question. I'm not sure that "context" is the good name. I need to have some global functions in the project. So I renamed it to "globalFunctions.js"? Is it okay? I need your feedback please.
However, the locale is already in the store with all translations.
The store.getState() method return this:
i18n{
    locale:"fr",
    translations:{
        de:{...},
        en:{...},
        fr:{...}
    }
}